eng/Version.Details.xml

```diff
@@ -24,5 +24,65 @@
+      <Uri>https://github.com/dotnet/runtime</Uri>
+      <Sha>ec3e0b276b159ab24946f17f2b89d00862488a6a</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.Extensions.Logging.Abstractions" Version="6.0.0-preview.5.21301.5" CoherentParentDependency="Microsoft.AspNetCore.App.Runtime.win-x64">
+      <Uri>https://github.com/dotnet/runtime</Uri>
+      <Sha>ec3e0b276b159ab24946f17f2b89d00862488a6a</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.Extensions.Logging" Version="6.0.0-preview.5.21301.5" CoherentParentDependency="Microsoft.AspNetCore.App.Runtime.win-x64">
+      <Uri>https://github.com/dotnet/runtime</Uri>
+      <Sha>ec3e0b276b159ab24946f17f2b89d00862488a6a</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.Extensions.Logging.Console" Version="6.0.0-preview.5.21301.5" CoherentParentDependency="Microsoft.AspNetCore.App.Runtime.win-x64">
+      <Uri>https://github.com/dotnet/runtime</Uri>
+      <Sha>ec3e0b276b159ab24946f17f2b89d00862488a6a</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.Extensions.Logging.Debug" Version="6.0.0-preview.5.21301.5" CoherentParentDependency="Microsoft.AspNetCore.App.Runtime.win-x64">
+      <Uri>https://github.com/dotnet/runtime</Uri>
+      <Sha>ec3e0b276b159ab24946f17f2b89d00862488a6a</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.Extensions.FileProviders.Embedded" Version="6.0.0-preview.5.21301.5" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
+      <Uri>https://github.com/dotnet/runtime</Uri>
+      <Sha>ec3e0b276b159ab24946f17f2b89d00862488a6a</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.AspNetCore.Authorization" Version="6.0.0-preview.5.21301.5" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
+      <Uri>https://github.com/dotnet/runtime</Uri>
+      <Sha>ec3e0b276b159ab24946f17f2b89d00862488a6a</Sha>
+    </Dependency>
+     <Dependency Name="Microsoft.AspNetCore.Components.WebView" Version="6.0.0-preview.5.21301.5" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
+      <Uri>https://github.com/dotnet/runtime</Uri>
+      <Sha>ec3e0b276b159ab24946f17f2b89d00862488a6a</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.AspNetCore.Components.Web" Version="6.0.0-preview.5.21301.5" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
+      <Uri>https://github.com/dotnet/runtime</Uri>
+      <Sha>ec3e0b276b159ab24946f17f2b89d00862488a6a</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.JSInterop" Version="6.0.0-preview.5.21301.5" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
+      <Uri>https://github.com/dotnet/runtime</Uri>
+      <Sha>ec3e0b276b159ab24946f17f2b89d00862488a6a</Sha>
+    </Dependency>
+      <Dependency Name="Microsoft.Web.WebView" Version="6.0.0-preview.5.21301.5" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
```

Comment:
Sorry I missed this earlier. This entry shouldn't be in here because it's not part of .NET 6.

Reply:
Fixed